myia/frontends/pytorch_functions.py

```diff
@@ -35,12 +35,44 @@
 # HELPER FUNCTIONS ##########################################################
 
 
+@macro
+async def _denom(info, shp_ref, dtype_ref, dim_ref, unbiased_ref):
```

Comment:
That's specifically the denominator for use in `var`, so I'd rename it `_var_denom` or something along these lines.

Also, `dtype_ref` is not used, and I believe this could be a `@myia_static`.